src/products/exceptions/get-all-products.exception.ts

```diff
@@ -0,0 +1,13 @@
+import { HttpException, HttpStatus, Logger } from '@nestjs/common';
```

Comment:
I suggest to chose a naming convention for Files, and stick to it in the whole project. Otherwise you are having camelCase in some filenames, and hypen-case in others.

Reply:
Why do you say camelCase in some filenames? I'm naming every file with hypen-case